doc/changelog/03-notations/11948-hexadecimal.rst

```diff
@@ -0,0 +1,12 @@
+- **Added:**
+  Numeral notations now parse hexadecimal constants such as ``0x2a``
+  or ``0xb.2ap-2``.  Parsers added for nat, positive, Z, N, Q, R,
```

Comment:
```suggestion
  or ``0xb.2ap-2``.  Parsers added for :g:`nat`, :g:`positive`, :g:`Z`, :g:`N`, :g:`Q`, :g:`R`,
```
(Though actually I'm not sure about the difference between :g: vs `` cc @coq/doc-maintainers )

Reply:
done